source/css/scss/app/pages/monthly-good.scss

```diff
@@ -148,6 +148,35 @@
 		}
 	}
 
+	.kiva-as-expert-section-wrapper {
+		background-color: $kiva-bg-lightgrey;
+
+		.kiva-as-expert-heading {
+			color: $kiva-green;
+			text-align: center;
+
+			@media #{$large-up} {
+				text-align: left;
+			}
+		}
+
+		.text-content {
+			@media #{$large-up} {
+				float: left;
```

Comment:
@BoulderBrains - not for a future refactor: would be nice if we could do this without floats.